src/main/java/de/tum/in/test/api/ast/type/ClassType.java

```diff
@@ -0,0 +1,53 @@
+package de.tum.in.test.api.ast.type;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.stmt.LocalClassDeclarationStmt;
+import com.github.javaparser.ast.stmt.LocalRecordDeclarationStmt;
+import com.github.javaparser.utils.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum ClassType {
+    /**
+     * Returns a list with all class types
+     */
+    ALL(new ArrayList<>() {
+        {
+            add(new Pair<>("Local-Class-Statement", LocalClassDeclarationStmt.class));
+            add(new Pair<>("Local-Record-Statement", LocalRecordDeclarationStmt.class));
+        }
+    }),
```

Comment:
[Implemented part of the feedback](https://github.com/ls1intum/Ares/pull/248/commits/7ed9fe2ac7f61d955c71ff99f56e550f8be89531): Use `Map.of` instead of `new ArrayList<>()`